### ISSUE TITLE
Crossfade should apply on chakra sounds and looping should terminate on inactivity

### ISSUE THREAD

**Issue by quinnouyang** [staylor7/rare#31]
Will likely need a basic custom looping: `setTimeout` with an off option if new category selected?

**Comment by quinnouyang** [comment]
On the side, should we always be looping even with no user activity? What about we only continue a loop given user activity? (e.g. no longer clicking anywhere -> stop when audio would end)

**Comment by staylor7** [comment]
Good idea! On Mar 17, 2024, at 4:06 PM, Quinn Ouyang ***@***.***> wrote:﻿
On the side, should we always be looping even with no user activity? What about we only continue a loop given user activity? (e.g. no longer clicking anywhere -> stop when audio would end)
